Makefile.toml

```diff
@@ -607,7 +607,7 @@ if [[ $ENABLE_COREDUMP == "true" ]]; then
 fi
 
 set -x
-target/${RISEDEV_BUILD_TARGET_DIR}${BUILD_MODE_DIR}/risingwave playground $@
+target/${RISEDEV_BUILD_TARGET_DIR}${BUILD_MODE_DIR}/risingwave --in-memory --max-idle-secs=1800 $@
```

Comment:
Is `playground` necessarily be in-memory? I think the persisted single-node mode is also good enough for local play.

Reply:
> I think the local-disk-based single-node mode is also good enough for local testing/playground purpose.

+1 for exposing local-disk based mode for our user by default.

> Moreover, we can remove the poorly-tested in-memory storage implementation.

I think it is better to keep the in-memory engine for now so that UTs can run without relying on local disk, which can be slower and requires post test clean-up. We can check whether we can switch to tempfs (RAM disk) if we really want to get rid of the in-memory engine.

Reply:
The only reason we may want to keep the `playground` alias is because of internal backwards compatibility. I think @mikechesterwang can just use `--in-memory` for this part, so we don't have to support `playground` alias, it will be messy.

Some users may just want to use an `--in-memory` version of risingwave, for playing around locally, so we can just keep this flag so we don't erase their usecase by accident.

This also allows us to continue supporting the web playground for risingwave.

Reply:
`./risingwave` will launch RisingWave with local file-system object store and SQLite meta store. It's already set to default. Only if user run `./risingwave playground` we well still use the in-memory mode. Honestly, I don't see a compelling reason to remove the in-memory playground mode.

Reply:
> `./risingwave` will launch RisingWave with local file-system object store and SQLite meta store. It's already set to default. Only if user run `./risingwave playground` we well still use the in-memory mode. Honestly, I don't see a compelling reason to remove the in-memory playground mode.

Make sense to me. My concern is that users may not realize the playground mode is completely volatile, but you are right, if they know nothing, they are probably using `./risingwave` (single-node) instead of playground.

Reply:
Well, regarding of [playground.risingwave.dev](https://playground.risingwave.dev/), I think the local-disk-based mode would be slightly better, because the in-memory storage is prone to OOM and may cause bad impression to users.

While for local `./risingwave playground`, either disk-based or in-memory LGTM. 

Overall, +1 for the motivation that unifies all of these "modes" into `single-mode`.